tasks/create-pyxis-image/create-pyxis-image.yaml

```diff
@@ -126,6 +126,9 @@ spec:
             DIGEST="${CONTAINER_IMAGE##*@}"
             PULLSPEC="${REPOSITORY}@${DIGEST}"
             MEDIA_TYPE=$(skopeo inspect --raw "docker://${PULLSPEC}" | jq -r .mediaType)
+            if [ $(jq ".components[${i}].tags | length" "${SNAPSHOT_SPEC_FILE}") -gt 0 ] ; then
+                TAGS=$(jq -r ".components[${i}].tags | join(\" \")" "${SNAPSHOT_SPEC_FILE}")
```

Comment:
Looking at how you do this in the populate-release-notes-images task where you have a separate variable made me look at this again. The concern is that you will override $TAGS if there are component tags, but what if the next component doesn't have any? The variable is wrong at that point. But it can probably never happen, because as soon as one component has tags, all of them will, right? WDYT?

Reply:
This is true, I am going to fix it

Reply:
fixed

Reply:
ok, cool